CMakeLists.txt

```diff
@@ -210,6 +218,15 @@ if(NOT MSVC)
   list(APPEND DGL_LINKER_LIBS metis)
 endif(NOT MSVC)
 
+# Compile LIBXSMM
+if(USE_LIBXSMM)
+  add_custom_target(libxsmm COMMAND make realclean COMMAND make -j BLAS=0
```

Comment:
The error is saying there's no `make` on the windows. I think you can just skip build LIBXSMM if it's on windows?

Reply:
What is the best way to check if it's on windows? Will "NOT MSVC" suffice?

Reply:
Yes. That's how we did now

Reply:
Ok great. Will do the same.

Reply:
It is building now, but it seems all the tests that are using "tests/compute/test_sparse.py" are failing. If there was some bug in the new SpMM code, then some of the tests would fail, not all. Since all the tests are failing, I am wondering if it is some setup issue. In the sense that the tests are not being built correctly. How do I run these tests? I tried ./runUnitTests at my end and all the tests passed.

Reply:
`runUnitTests` runs the C++ unit tests, and indeed they are all correct according to CI since you don't have new C++ unit tests (in `tests/cpp`).  The unit tests are failing on Python side, which needs `pytest` to run:
```
# Assuming that your current directory is the Git root directory
export PYTHONPATH=$PWD/tests
pytest tests/compute/test_sparse.py
# Or you can run pytest with Python builtin debugger, though I'm not sure if that will help for C++ debugging
pytest --pdb tests/compute/test_sparse.py
```